tests/by-util/test_kill.rs

```diff
@@ -334,6 +334,15 @@ fn test_kill_with_signal_and_list() {
         .fails();
 }
 
+#[test]
+fn test_kill_with_list_lower_bits() {
+    new_ucmd!()
+        .arg("-l")
+        .arg("143")
+        .succeeds()
+        .stdout_matches(&Regex::new("TERM").unwrap());
```

Comment:
While this works, I think using a regex is a bit overkill and you can simplify it by using either `stdout_is` or `stdout_contains`.

Reply:
Yes, that's also fine.